issue/controller.py

```diff
@@ -0,0 +1,68 @@
+import requests
+import xmltodict
+import json
+
+from .models import ScieloJournal, Issue
+from processing_errors.models import ProcessingError
+
+
+def get_journal_xml(collection, issn):
+    try:
+        journal = requests.get(
+            f"http://{collection}/scielo.php?script=sci_issues&pid={issn}&lng=es&nrm=iso&debug=xml", timeout=10
+        )
+
+        return xmltodict.parse(journal.text)
+
+    except Exception as e:
+        error = ProcessingError()
+        error.item = f"Error getting the ISSN {issn} of the {collection} collection"
+        error.step = "Journal record search error"
+        error.description = str(e)[:509]
+        error.type = str(type(e))
+        error.save()
+
+
+def get_issue(user, journal_xml):
+    issn_scielo = journal_xml['SERIAL']['ISSN_AS_ID']
+    try:
+        journal = ScieloJournal.objects.filter(issn_scielo=issn_scielo)[0]
+        for issue in journal_xml['SERIAL']['AVAILISSUES']['YEARISSUE']:
+                volume = issue['VOLISSUE']['@VOL']
```

Comment:
@Rossi-Luciano confere a carga com Chile, pois sei que lá há casos em que não há volume. E outra coisa, neste caso pode vir a ocorrer uma exceção que não está sendo tratada, pois o try está somente no for de dentro.

Reply:
@robertatakenaka feito.

Reply:
@Rossi-Luciano então, mas como ficou a carga do Chile. Não deve levantar exceção, porque não é exceção. O programa tem que lidar quando o Volume é ausente